code/modules/mob/living/basic/farm_animals/chickens.dm

```diff
@@ -0,0 +1,142 @@
+/mob/living/basic/chick
+	name = "\improper chick"
+	desc = "Adorable! They make such a racket though."
+	icon_state = "chick"
+	icon_living = "chick"
+	icon_dead = "chick_dead"
+	icon_gib = "chick_gib"
+	gender = FEMALE
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	density = FALSE
+	speak_emote = list("cheeps")
+	speed = 1.5
+	butcher_results = list(/obj/item/food/meat/slab/chicken = 1)
+	response_help_continuous = "pets"
+	response_help_simple = "pet"
+	response_disarm_continuous = "gently pushes aside"
+	response_disarm_simple = "gently push aside"
+	response_harm_continuous = "kicks"
+	response_harm_simple = "kick"
+	attack_verb_continuous = "kicks"
+	attack_verb_simple = "kick"
+	health = 3
+	maxHealth = 3
+	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
+	mob_size = MOB_SIZE_TINY
+	gold_core_spawnable = FRIENDLY_SPAWN
+	footstep_type = FOOTSTEP_MOB_CLAW
+
+	ai_controller = /datum/ai_controller/basic_controller/chick
+
+/mob/living/basic/chick/Initialize()
+	. = ..()
+	AddElement(/datum/element/pet_bonus, "chirps!")
+	AddElement(/datum/element/swabable, CELL_LINE_TABLE_CHICKEN, CELL_VIRUS_TABLE_GENERIC_MOB, 1, 5)
+	pixel_x = base_pixel_x + rand(-6, 6)
+	pixel_y = base_pixel_y + rand(0, 10)
+	ADD_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+	addtimer(CALLBACK(src, .proc/hologram_check), 1)
+
+///waits until the next tick for this check as we won't know if we're holographic on init
+/mob/living/basic/chick/proc/hologram_check()
+	if(!(flags_1 & HOLOGRAM_1))
+		AddComponent(/datum/component/growing_up, /mob/living/basic/chicken, 100)
+
+/datum/ai_controller/basic_controller/chick
+
+	ai_traits = STOP_MOVING_WHEN_PULLED
+	ai_movement = /datum/ai_movement/basic_avoidance
+
+	//IMPLEMENT IDLE BEHAVIOR HERE
```

Comment:
requires https://github.com/tgstation/tgstation/pull/61455,

will allow for a generic idle datum